client/src/Components/AddMedicine/index.js

```diff
@@ -1,129 +1,123 @@
-import React, { Component } from 'react'
-import NavBar from './../NavBar'
-import './style.css'
-import axios from 'axios'
+import React, { Component } from "react"
+import NavBar from "./../NavBar"
+import "./style.css"
+import axios from "axios"
 
 class AddMedicine extends Component {
-	state = {
-		medName: '',
-		medPrice: '',
-		medCompany: '',
-		prescription: false,
-		msg: '',
-		submitClicked: false,
-		login: false
-	}
+  state = {
+    medName: "",
+    medPrice: "",
+    medCompany: "",
+    prescription: false,
+    msg: "",
+    submitClicked: false,
+    login: false
+  }
 
-	componentDidMount = () => {
-		const { history } = this.props
+  componentDidMount = () => {
+    const { history } = this.props
 
-		axios.get('/check-auth').then(({ data }) => {
-			const { success } = data
-			if (success) {
-				this.setState({
-					login: true
-				})
-			} else {
-				history.push('/login')
-			}
-		})
-	}
+    axios.get("/check-auth").then(({ data }) => {
+      const { success } = data
+      if (success) {
+        this.setState({
+          login: true
+        })
+      } else {
+        history.push("/login")
+      }
+    })
+  }
 
-	updateInput = event => {
-		const { value, name } = event.target
-		this.setState({ [name]: value, submitClicked: false })
-	}
+  updateInput = event => {
+    const { value, name } = event.target
+    this.setState({ [name]: value, submitClicked: false })
+  }
 
-	toggleCheckbox = () => {
-		this.setState({ prescription: !this.state.prescription })
-	}
+  toggleCheckbox = () => {
+    this.setState({ prescription: !this.state.prescription })
+  }
 
-	AddMedicineInfo = event => {
-		event.preventDefault()
-		const { medName, medCompany, medPrice, prescription } = this.state
-		this.setState({ submitClicked: true })
-		if (medName === '' || medCompany === '' || medPrice === '') {
-			return
-		}
+  AddMedicineInfo = event => {
+    event.preventDefault()
+    const { medName, medCompany, medPrice, prescription } = this.state
+    this.setState({ submitClicked: true })
+    if (medName === "" || medCompany === "" || medPrice === "") {
+      return
+    }
 
-		axios
-			.post('/api/pharmacy/medicine', {
-				medName,
-				medCompany,
-				prescription,
-				medPrice
-			})
-			.then(res => {
-				if (res.data.message == 'true') {
-					this.setState({ msg: 'true' })
-				}
-			})
-	}
+    axios
+      .post("/api/pharmacy/medicine", {
+        medName,
+        medCompany,
+        prescription,
+        medPrice
+      })
+      .then(res => {
+        const { add, message } = res.data
+        if (add && message == "done") {
+          this.setState({ msg: message })
+        } else {
+          this.setState({ msg: message })
+        }
```

Comment:
why you used if statement if you already doing the same thing in both conditions